scripts/sass-to-lit-css/index.js

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env node
+
+/**
+@license
```

Comment:
mind just opening a quick issue / task to migrate all of our license headers to the new, shorter format?

Reply:
Filed as #2419. I'll update this template to use the SPDX header, and follow up with the rest of the code in a separate PR.